src/components/Alert/Alert.stories.js

```diff
@@ -1,19 +1,19 @@
-import Alert from './Alert.svelte';
+import Alert from './Alert';
 
 export default { title: 'Alert' };
 
 export const errorWithText = () => ({
   Component: Alert,
   props: {
     type: 'error',
-    text: 'Virhe',
-  },
+    text: 'Virhe'
+  }
 });
 
 export const successWithText = () => ({
   Component: Alert,
   props: {
     type: 'success',
-    text: 'Onnistu',
-  },
+    text: 'Onnistu'
```

Comment:
Should it say something else except "Onnistu"?

Reply:
Story-file, so its only dummydata for storybook.